src/main/java/org/rumbledb/expressions/primary/PrimaryExpression.java

```diff
@@ -1,45 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * Authors: Stefan Irimescu, Can Berker Cikis
- *
- */
-
-package org.rumbledb.expressions.primary;
-
-
-import org.rumbledb.exceptions.ExceptionMetadata;
-import org.rumbledb.expressions.Expression;
-import org.rumbledb.expressions.Node;
-
-import java.util.ArrayList;
-import java.util.List;
-
-
-// TODO(NOT IMPLEMENTED): orderedExpr, unorderedExpr
```

Comment:
Would you like to create an issue for this removed TODO: item, or is this already within our development plans?

Reply:
I have somewhere a masterplan of missing features, i.e., this is under control. Thanks for asking.